lm_eval/tasks/selfcheckgpt/selfcheckgpt.yaml

```diff
@@ -0,0 +1,28 @@
+task: selfcheckgpt
+dataset_path: potsawee/wiki_bio_gpt3_hallucination
+output_type: generate_until
+training_split: null
+validation_split: evaluation
+doc_to_text: !function utils.doc_to_text
+doc_to_target: !function utils.doc_to_target
+process_results: !function utils.process_results
+generation_kwargs:
+  # until:
+  #   - "\n"
+  temperature: 0.0
+  do_sample: false
+generation_kwargs_sampling_number: 5
```

Comment:
`repeats: 5` is already supported to enable multiple generations per document, so we don't need to add `generation_kwargs_sampling_number` as a potential arg!

Reply:
Also, when using `repeats: N` to get different non-deterministic generations, I think `sampling` in `generation_kwargs` should be set to `True`.